java/ingest/src/test/java/sleeper/ingest/job/IngestJobRunnerTest.java

```diff
@@ -217,16 +210,15 @@ private void consumeAndVerify(Schema sleeperSchema,
         stateStore.initialise();
 
         // Run the job consumer
-        IngestJobQueueConsumer ingestJobQueueConsumer = new IngestJobQueueConsumer(
+        IngestJobRunner ingestJobQueueConsumer = new IngestJobRunner(
                 new ObjectFactory(instanceProperties, null, temporaryFolder.newFolder().getAbsolutePath()),
-                AWS_EXTERNAL_RESOURCE.getSqsClient(),
-                AWS_EXTERNAL_RESOURCE.getCloudWatchClient(),
                 instanceProperties,
                 tablePropertiesProvider,
                 stateStoreProvider,
                 localDir,
                 AWS_EXTERNAL_RESOURCE.getS3AsyncClient(),
-                AWS_EXTERNAL_RESOURCE.getHadoopConfiguration());
+                AWS_EXTERNAL_RESOURCE.getHadoopConfiguration(),
+                ingestJobList);
         ingestJobQueueConsumer.run();
```

Comment:
We probably only need to test a single ingest job in each test, since taking multiple jobs off the queue will be tested against IngestJobQueueConsumer.